engine/src/main/java/org/destinationsol/common/SolMath.java

```diff
@@ -205,15 +207,15 @@ public static Vector2 fromAl(float angle, float len, boolean precise) {
      * @return a new bound copy of src
      */
     @Bound
-    public static Vector2 getVec(Vector2 src) {
+    public static @NotNull Vector2 getVec(@NotNull Vector2 src) {
```

Comment:
I think we can remove the NotNull, and let an NPE speak for itself.

Reply:
All @NotNull/@Nullable annotations will get eventually removed